source/configuration_manual/authentication/dict.rst

```diff
@@ -257,8 +257,7 @@ Complete example for authenticating via a UNIX socket
 
 The Dict auth backend can be used to query a local UNIX socket for users. This
 can be handy for accessing user databases which would otherwise only be
-accessible via the :ref:`authentication-checkpassword` backend and a scripting
-language.
+accessible by a scripting language.
```

Comment:
I think "accessible by Lua." with a link to passdb-lua.

Reply:
this talks about using UNIX sockets. So there is no natural place to mention Lua here, unless I add .. warning or .. note?

Reply:
If you read the whole text, it's talking about a background of how this provides access to a database that wouldn't be otherwise accessible by Dovecot, except by passdb/userdb lua. But then again the whole paragraph seems rather confusing and pointless, so might as well remove it entirely?

Reply:
I think the original text intended to say it's only accessible with checkpassword script and the "scripting language" meant the checkpassword script..

Reply:
```
The Dict auth backend can be used to query a local UNIX socket for users, but you
should consider using :ref:`authentication-lua_based_authentication` instead.
```
?